packages/astro/src/@types/astro.ts

```diff
@@ -1087,9 +1087,13 @@ export interface AstroUserConfig {
 	 * }
 	 * ```
 	 */
-	integrations?: Array<
-		AstroIntegration | (AstroIntegration | false | undefined | null)[] | false | undefined | null
-	>;
+	integrations?: (
+		| AstroIntegration
+		| (AstroIntegration | false | undefined | null)[]
+		| false
+		| undefined
+		| null
+	)[];
```

Comment:
Switching from `Array<...>` to `(...)[]` is very much a _`stylistic`_ choice (https://typescript-eslint.io/rules/array-type/). It seemed to me like `[]` was generally preferred already?

Reply:
I usually use `(...)[]` so this is fine by me.

Reply:
I'd argue against linting for either. `(...)[]` looks neat with smaller types but becomes easy to miss with larger unions and object types.

Reply:
@lilnasy does https://typescript-eslint.io/rules/array-type/#array-simple match your preference?

Reply:
That's perfect!

Reply:
I'm not really sure about `array-simple`. If we're undecided about this, I think we can also leave it unchanged for now (sorry for causing more changes)

Reply:
No worries 😄 I'll go through and undo the changes, then merge from `main` & re-request your review. Thanks!

Reply:
Done. This is a lesson learned the hard way - when there are many complaint from a stylistic rule, leave it out in the proposal. 😄